installer.sha

```diff
@@ -1 +1 @@
-6bf329f3ff088fa56e537afecb31938f49800978
\ No newline at end of file
+a4241599ca243b2cc6bdfd28fb45c26709143438
```

Comment:
This sha is a _master_ sha. You'll want to use a release-1.4 sha. I and @sdake tried using a6e4a4b3087ffd1bce5b186c0e3bd49cf2ee40a0 yesterday (latest one), but ran into mandiff errors we hoped this cherry pick would fix.

Reply:
I pushed a new commit with sha changed to a6e4a4b3087ffd1bce5b186c0e3bd49cf2ee40a0 and
"make update-goldens".

Reply:
OK. So that looks good.